ethexe/processor/src/handling/mod.rs

```diff
@@ -31,6 +32,7 @@ pub struct ProcessingHandler {
     pub block_hash: H256,
     pub db: Database,
     pub transitions: InBlockTransitions,
+    pub dispatch_origin: Origin,
```

Comment:
shouldn't be here at all, only where we process journal

Reply:
Fixed